stock_valued_picking_report/README.rst

```diff
@@ -0,0 +1,77 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+Stock Valued Picking Report
+===========================
+
+Add amount information to Delivery Slip report.
+You can select at partner level if picking list report must be valued or not.
+
+Configuration
+=============
+
+#. Go to *Customers > (select one of your choice) > Sales & Purchases*.
+#. Set *Valued picking* field on.
```

Comment:
I'm testing in runbot but @chienandalu I cannot find this option even with developer mode activated and all permissions, check:
![2017-09-05_2-15-21](https://user-images.githubusercontent.com/8736623/30040512-6d6ddf4e-91e0-11e7-986d-402d08a6f2f8.png)

Reply:
OK, this is because ASUSTek is not a customer